runtime/tools/python/ttrt/common/api.py

```diff
@@ -178,13 +178,14 @@ def run(args):
 
     # execution
     print("executing action for all provided flatbuffers")
-    device = ttrt.runtime.open_device(device_ids)
+    # TODO: sort the flatbuffers by runtime type, for now just take the first one
```

Comment:
@tapspatel, can you file an issue to track this?  An alternative is that you're forced to just select which runtime to use when you launch `ttrt run`.  Or we could sort the flatbuffers by runtime type so we open a ttnn device once, run all the ttnn tests, close it, then open a ttmetal device once run all the ttmetal tests and then close it.